model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -167,13 +175,16 @@ data class PackageCurationData(
      * Merge with [other] curation data. The `comment` properties are joined but probably need to be adjusted before
      * further processing as their meaning might have been distorted by the merge. For properties that cannot be merged,
      * data in this instance has precedence over data in the other instance.
+     *
+     * TODO: Clarify semantics if setting an author os copyright holders replaces or augments any scan result value
```

Comment:
"TODO: Clarify semantics about whether setting an author or copyright holders should replace or amend results from a scan."